src/android/InAppBrowser.java

```diff
@@ -147,6 +148,7 @@ Licensed to the Apache Software Foundation (ASF) under one
     private boolean showFooter = false;
     private String footerColor = "";
     private String beforeload = "";
+    private boolean fullscreen = false;
```

Comment:
See above, the default need to be true in order to avoid a breaking change.

Reply:
I agree. I set it to false as the web view should have never been opened in full screen in the first place (unless it's a game). Anyway, I understand the backward compatibility requirement.

Reply:
@breautek @NiklasMerz - what's your opinion on this? It seems v3.1.0 accidentally sneaked in this breaking change (see, e.g. https://github.com/apache/cordova-plugin-inappbrowser/issues/598#issuecomment-572597994)

Should we (a) restore v3.0.0 behavior or (b) follow semver and make it configurable in a minor release?

The change to always full screen is now live since Jun 28, 2019, so nearly two years.

Reply:
Not very knowledgeable with the in-app-browser but unless if there is a very good reason why it should be hard-coded (which I don't think there is any?), I think we should be non-opinionated and let it be configurable.

Normally I'd also vote to revert the default as well, but if it's been live for nearly two years, I think it's smarter to just leave the "default" as `true`. I think if we change it to `false` now, then we would have to treat this PR as a breaking change.